build.gradle

```diff
@@ -787,6 +793,12 @@ project(":samza-test_$scalaVersion") {
   sourceSets.main.java.srcDirs = []
   sourceSets.test.java.srcDirs = []
 
+  // Disabling assertions while running test because of a calcite bug in {@link RelOptUtil#eq} method
```

Comment:
Is this a known issue in Calcite ? Could we have the ticket here so that we could enable the assertions when it is fixed ?

Reply:
Yup it's a bug in calcite code in RelOptUtil. Opened https://issues.apache.org/jira/browse/SAMZA-2069 to track this.